src/OpenTracing/Log/Fields.cs

```diff
@@ -0,0 +1,57 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Fields.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing.Log
+{
+    /// <summary>
+    ///     The following log fields are recommended for instrumentors who are trying to capture more
+    ///     information about a logged event. Tracers may expose additional features based on these
+    ///     standardized data points.
+    ///     <see href="https://github.com/opentracing/specification/blob/master/semantic_conventions.md" />
+    /// </summary>
+    public static class Fields
+    {
+        /// <summary>
+        ///     The type or "kind" of an error (only for event="error" logs). E.g., "Exception", "OSError"
+        /// </summary>
+        public static readonly string ErrorKind = "error.kind";
```

Comment:
I think these things should stay constants as they will never change at runtime and therefore avoid a method call.

Reply:
No method call for a field, an JIT should inline it anyway, but yes, for these that won't change permitting COMPILER inlining like consts do would be good.

Reply:
resolved